demo/install_cli.sh

```diff
@@ -7,7 +7,7 @@ function find_python() {
   local python_candidate
 
   while read python_candidate; do
-    if [[ $("$python_candidate" --version 2>/dev/null) =~ (^|[^[:digit:].])3.10(\.|$) ]]; then
+    if [[ $("$python_candidate" --version 2>/dev/null) =~ (^|[^[:digit:].])3.[7,8,9,10,11](\.|$) ]]; then
```

Comment:
This isn't quite right, but neither is the original:
```
% [[ "Python 3.17.3" =~ (^|[^[:digit:].])3.[7,8,9,10,11](\.|$) ]] && echo ehhhhh
ehhhhh
```
Instead, I think the pattern should be,
```
(^|[^[:digit:].])3\\.(7|8|9|10|11)(\\.|$)
```

Reply:
nm. 1 back slash is sufficient:
```
(^|[^[:digit:].])3\.(7|8|9|10|11)(\.|$)
```